packages/geo/src/lib/measure/measurer/measurer.component.ts

```diff
@@ -527,7 +523,12 @@ export class MeasurerComponent implements OnInit, OnDestroy {
    */
   onAreaUnitChange(unit: MeasureAreaUnit) {
     this.activeAreaUnit = unit;
-    this.table.refresh();
```

Comment:
@aziz-access J'ai supprimé le `table.refresh()`. En théorie, avec le changement que tu as fait on ne devrait plus avoir besoin de l'invoquer. Good for you?

Reply:
ok good